shared/reducers/chat.js

```diff
@@ -79,6 +79,16 @@ function reducer (state: State = initialState, action: Actions) {
   switch (action.type) {
     case CommonConstants.resetStore:
       return initialState
+    case 'chat:removeOutboxMessage': {
```

Comment:
Cool.  @chrisnojima, what do you want to do?  I feel like adding it to deletedIDs would be the incorrect thing here, because it would be making a claim that it was deleted (if we ever receive it again) when it actually wasn't at all.